### DISCUSSION TITLE
How to change the distance between questions in likert survey?

### DISCUSSION THREAD

**Discussion by kkcs2000** [jspsych/jsPsych#1323] [Q&A]
Hi, 
I am now preparing a likert survey, with more than one questions in the same page.
The distance between questions seems too far with each other. Is it possible to change the distance between questions?
I checked the jspsych-survey-likert.js plugin and jspsych.css, and did not figure out how to to this.
Your advice will be highly appreciated! Thank you.

**Comment by jodeleeuw** [comment]
Hi @kkcs2000,

You could adjust the padding-top property here, which is the main CSS rule creating space between the items.

https://github.com/jspsych/jsPsych/blob/master/plugins/jspsych-survey-likert.js#L95

**Reply by kkcs2000** [reply]
Thank you for the prompt reply!
I tried to set the padding-top to 0px and even minus value, but there is still quite a gap between items. 
I think this is the best we can do. Thank you!

**Reply by jodeleeuw** [reply]
Can you post a screenshot of what it looks like for you?

**Reply by kkcs2000** [reply]
![スクリーンショット 2020-12-21 2 55 52](https://user-images.githubusercontent.com/76264595/102720529-37e64700-4338-11eb-889a-9ede844b0719.jpg)
Thank you!
This is the snapshot when I set padding-top to 0px. Much better than 40px, the default value, but still a gap.

**Reply by kkcs2000** [reply]
BTW, 40px is like this.
![スクリーンショット 2020-12-21 2 54 32](https://user-images.githubusercontent.com/76264595/102720578-97dced80-4338-11eb-907e-d3a3e6c114ed.jpg)

**Reply by jodeleeuw** [reply]
Thanks, this is very helpful. We designed the CSS with the use of the `prompt` parameter in mind. Since you don't have a `prompt` there is a larger gap than intended. You could remove the `margin-bottom` rule on the same line I linked above. I think that will effectively remove the space that was used for the prompt.

**Reply by kkcs2000** [reply]
Thank you for the reply!
I deleted the `margin-bottom` rule and the line was changed as follows:
`html += ".jspsych-survey-likert-statement { display:block; font-size: 16px; padding-top: 0px; }"+`
However, it doesn't work.

I think the space for the prompt was not displayed when I used `prompt: ""`(Fig 1 below). Because when I added some prompt for test, the space became larger (Fig 2 below). The space for the line of prompt in Fig 2 was omitted in Fig 1.

![スクリーンショット 2020-12-22 2 56 54](https://user-images.githubusercontent.com/76264595/102807536-2f5a4300-4402-11eb-8429-dba68556fede.jpg)
(Fig 1)

![スクリーンショット 2020-12-22 2 55 50](https://user-images.githubusercontent.com/76264595/102807555-38e3ab00-4402-11eb-94eb-7fbcdbc80120.jpg)
(Fig 2)
